src/main/java/org/jabref/gui/consistency/ConsistencySymbolsDialog.java

```diff
@@ -0,0 +1,21 @@
+package org.jabref.gui.consistency;
+
+import javafx.stage.Modality;
+
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.logic.l10n.Localization;
+
+import com.airhacks.afterburner.views.ViewLoader;
+
+public class ConsistencySymbolsDialog extends BaseDialog<Void> {
+
+    public ConsistencySymbolsDialog() {
+        this.setTitle(Localization.lang("Symbols information"));
+        this.initModality(Modality.NONE);
+        this.setResizable(false);
+
+        ViewLoader.view(this)
+                  .load()
+                  .setAsDialogPane(this);
+    }
+}
```

Comment:
Is there a real use case a user would want to customize the symbols? Everything needs to be maintained, forever.
I would just stick to the symbols you or @koppor already decided for.

Reply:
oh sorry, i misinterpreted the dialog. Forget this comment.